### PR TITLE
Update image-accessible-name-descriptive-qt1vmo.md

### PR DESCRIPTION
**Pull request by wcreedle**
Added description for failed example 2

<< Describe the changes >>

Need for Call for Review:

This will not require a Call for Review . 

---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by WilcoFiers — APPROVED** [review]

**Review by HelenBurge — APPROVED** [review]
Might be good to amend the PR default text as you should add descriptions of what you did and why!

**Review by tbostic32 — APPROVED** [review]

File: _rules/image-accessible-name-descriptive-qt1vmo.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ This `img` element has an `alt` attribute that incorrectly describes the image.
 
 #### Failed Example 2
 
-This `svg` element has an `aria-label` attribute that incorrectly describes the image.
+This `svg` element has an `aria-label` attribute that incorrectly describes the image (aria-label describes W3C Logo but the actual image is HTML5 logo).
```
Comment (Jym77) [inline _rules/image-accessible-name-descriptive-qt1vmo.md]:
```suggestion
This `svg` element has an `aria-label` attribute that incorrectly describes the image (the `aria-label` is "W3C" but the actual image is the HTML5 logo).
```

File: _rules/image-accessible-name-descriptive-qt1vmo.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ This `svg` element has an `aria-label` attribute that incorrectly describes the
 
 #### Failed Example 3
 
-This `canvas` element has an `aria-label` attribute that incorrectly describes the image.
+This `canvas` element has an `aria-label` attribute that incorrectly describes the image (aria-label describes HTML5 logo but the actual image is W3C logo).
```
Comment (Jym77) [inline _rules/image-accessible-name-descriptive-qt1vmo.md]:
```suggestion
This `canvas` element has an `aria-label` attribute that incorrectly describes the image (the `aria-label` is "HTML5 logo" but the actual image is the W3C logo).
```

**Review by Jym77 — DISMISSED** [review]
Small polishing.

**Review by Jym77 — APPROVED** [review]

**Pull request merged**